Fix playground IDs in car animation documentation (#1597)
BabylonJS/Documentation: content/How_To/camera/WebVR_Camera.md

Note that this will create a new quaternion to the mesh .

> The default controller mesh can be disappeared by the following code.

Reviewer: ```suggestion
The default controller mesh can be hidden with the following code.
```

Reviewer 2: Thanks for fixing the grammatical mistake.

Reviewer: You did not apply my suggestion:)
